src/interface/src/app/plan/delete-note-dialog/delete-note-dialog.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+import { MatLegacyDialogRef as MatDialogRef } from '@angular/material/legacy-dialog';
+
+@Component({
+  selector: 'app-delete-note-dialog',
+  templateUrl: './delete-note-dialog.component.html',
+  styleUrl: './delete-note-dialog.component.scss',
+})
+export class DeleteNoteDialogComponent {
+  disableDeleteButton: boolean = false;
+  error: any;
```

Comment:
are we using this? can we add types?

Reply:
we're not. I was originally, but yeah, it's cruft.